healthmap/mriids2
# Remove local JSON files as they are now fetched by API Call

Issue fixed: http://jira/browse/MRIID-83 ( Remove local JSON files as they are now fetched by API Call)

Test Case
- Pull down the branch
- Start the server
- Double check in network tab that JSON's are fetched via API calls.
- Make sure the site is working as expected.
